File: expressions/financial_calculator.py
#Gabriel Lopez, Financial Calculator on Python

# print statement that welcomes user and tells what program does

# ask what their income is (variable an input)
income = float(input("What is your monthly income?\n"))
# ask what their rent is (variable an input)
rent = float(input("What is your monthly rent?\n"))
# ask what their utilities is (variable an input)
utilites = float(input("What is your monthly expense for utilities?\n"))
# ask what their groceries is (variable an input)
groceries = float(input("What is your monthly expense for groceries?\n"))
# ask what their transportation is (variable an input)
transportation = float(input("What is your monthly expense for transportation?\n"))
# calculate savings as 10% of income (income*.1) (variable)
savings = (income/10)
# calculate spending as income-savings-rent-utilities-groceries-transportation (variable)
spending = (income - (rent + utilites + groceries + transportation))
# calculate percent income of rent (rent/income *100) (variable)
income_of_rent = (rent/income *100)
# calculate percent income of utilities (utilities/income *100) (variable)
income_of_utilities = (utilites/income *100)
# calculate percent income of groceries (groceries/income *100) (variable)
income_of_groceries = (groceries/income *100)
# calculate percent income of transportation (transportation/income *100) (variable)

# calculate percent income of spending (spending/income *100) (variable)

# Your rent is $XX.XX which is XX% of your income. (Print)

# Your utlities is $XX.XX which is XX% of your income. (Print)

# Your groceries is $XX.XX which is XX% of your income. (Print)

# Your transportation is $XX.XX which is XX% of your income. (Print)

# Your savings is $XX.XX which is XX% of your income. (Print)

# Your spendings is $XX.XX which is XX% of your income. (Print)

File: expressions/financial_calculator.c
//Gabriel Lopez, Financial Calculator on C
#include <stdio.h>
#include <math.h>

int main(void){
    float income, rent, utilities, groceries, transportation, savings, expenses, total; 
    float iorent, ioutilities, iogroceries, iotransportation, ioexpenses, iosavings;
    printf("Welcome to the Financial Calculator!\n");
    printf("How much do you make monthly?");
    scanf("%f", &income);
    printf("What is your monthly rent?");
    scanf("%f", &rent);
    printf("What is your monthly payment for utilities?");
    scanf("%f", &utilities);
    printf("What is your monthly payment for groceries");
    scanf("%f", &groceries);
    printf("What is your monthly payment for transportation?");
    scanf("%f", &transportation);
    expenses = rent + utilities + groceries + transportation;
    savings = income * .2;
    total = income - savings - expenses;
    printf("Your monthly income is $%.2f\n", income);
    printf("Your monthly expenses are $%.2f\n", expenses);
    printf("Your monthly savings is $%.2f\n", expenses);
    printf("You have $%.2f\n", total);
    iorent = rent/income *100;
    ioutilities = utilities/income *100;
    iogroceries = groceries/income *100;
    iotransportation = transportation/income *100;
    ioexpenses = expenses/income *100;
    iosavings = savings/income *100;
    printf("Your expenses are %.1f%% of your income\n", ioexpenses);
    printf("Your savings are %.1f%% of your income\n", iosavings);
    return 0;
}